pages/common/doggo.md

```diff
@@ -0,0 +1,25 @@
+# doggo
+
+> Command-line DNS client for Humans.
+> Written in Golang.
+> More information: <https://doggo.mrkaran.dev>.
+
+- Simple DNS lookup:
+
+`doggo {{example.com}}`
+
+- Query MX records using a specific nameserver:
+
+`doggo MX {{codeberg.org}} @{{1.1.1.2}}`
+
+- Use DNS over HTTPS:
+
+`doggo {{example.com}} @{{https://dns.quad9.net/dns-query}}`
+
+- JSON output for scripting:
```

Comment:
```suggestion
- Output in the JSON format:
```
See https://github.com/tldr-pages/tldr/blob/main/contributing-guides%2Fstyle-guide.md#imperative-mood. Can you please update other examples?

Reply:
Can do.